airflow/dags/monitor/monitor.py

```diff
@@ -22,6 +27,11 @@
 
 google_service_account_json_value = os.environ.get("GOOGLE_APPLICATION_CREDENTIALS_VALUE", None)
 
+ASK_ASTRO_API_BASE_URL = os.environ.get("ASK_ASTRO_API_BASE_URL", None)
+
+ASK_ASTRO_REQUEST_ID_1 = os.environ.get("ASK_ASTRO_REQUEST_ID_1", "05d8882e-56ac-11ee-a818-4200a9fe0102")
```

Comment:
are these id valid ones of example ones? if they're example ones, could we add a comment here?

Reply:
https://github.com/astronomer/ask-astro/pull/98/commits/63cf506948703342764c45d64789412aae4fe0f9